provisioning/osc-provision

```diff
@@ -201,6 +211,20 @@ provision_nodes() {
   $command || error_out "Node Provision failed. See log at ${LOGFILE}." ${ERROR_CODE_PROVISION_FAILURE}
 }
 
+sync_files() {
+  # Sync files from variable with format "file1:dest1 file2:dest2"
+  files=$1
+  nodes=$2
+
+  for node in $nodes; do
+    for file in $files; do
+      source=${file%:*}
+      dest=${file#*:}
+      ${SCP_CMD} ${source} root@${node}:${dest}
+    done
```

Comment:
This may be good for now, but really need also think about setting file permissions, ownership and SELinux context on the dest side.